### PR TITLE
Fix support for `open import foundation` and an automatic line wrap misplacement

### PR DESCRIPTION
**Pull request by fredrik-bakke**
This PR fixes two bugs related to our autoformatting scripts for Agda code
1. `fix_imports.py` no longer crashes in the presence of `open import foundation` if there are no `foundation-core` imports
2. Too long lines of the form `mydef A B C =` are now correctly wrapped right after `mydef`

### CODE REVIEW

**Comment by fredrik-bakke** [comment]
Turns out `open import group-theory` was always valid. The issue was a particular one when importing `foundation`. @EgbertRijke

**Comment by fredrik-bakke** [comment]
This PR is ready for review now

**Comment by EgbertRijke** [comment]
Great! I will have a look when I get back to the hotel

File: scripts/wrap_long_lines_simple.py [hunk 1]
```diff
@@ -73,8 +73,9 @@ def check_wrap_line_definition_parameters(line):
     and if so add a line wrap after the first token.
     """
     tle = get_top_level_equality(line)
-    if tle and tle == len(line) - 1:
-        m = re.fullmatch(r'((\s*)[^\s.;{}()@"]+)\s+([^.;()@"]+\s+=)', line)
+
+    if tle and tle >= len(line) - 1:
+        m = re.fullmatch(r'((\s*)[^\s.;{}()@"]+)\s+([^.;()@"]+\s+=)\s*', line)
```
Comment (fredrik-bakke) [inline scripts/wrap_long_lines_simple.py:78]:
```suggestion
    if tle and tle == len(line) - 1:
        m = re.fullmatch(r'((\s*)[^\s.;{}()@"]+)\s+([^.;()@"]+\s+=)', line)
```

**Comment by EgbertRijke** [comment]
LGTM

**Pull request merged**